api/application.py

```diff
@@ -19,9 +20,14 @@
 from notd.store.saver import Saver
 from notd.token_client import TokenClient
 
+from fastapi.exceptions import RequestValidationError
+from fastapi.responses import JSONResponse
+
 logging.basicConfig(level=logging.INFO)
 
-database = Database(f'postgresql://{os.environ["DB_USERNAME"]}:{os.environ["DB_PASSWORD"]}@{os.environ["DB_HOST"]}:{os.environ["DB_PORT"]}/{os.environ["DB_NAME"]}')
+#database = Database(f'postgresql://{os.environ["REMOTE_DB_USERNAME"]}:{os.environ["REMOTE_DB_PASSWORD"]}@{os.environ["REMOTE_DB_HOST"]}:{os.environ["REMOTE_DB_PORT"]}/{os.environ["REMOTE_DB_NAME"]}')
```

Comment:
none of the changes in this file should be committed (read about how to not commit all files by adding specific ones to a commit)